torch/_higher_order_ops/wrap.py

```diff
@@ -9,5 +10,23 @@ def __call__(self, func, *args):
         result = func(*args)
         return result
 
-
 wrap = Wrap()
+
+class WrapActivationCheckpoint(HigherOrderOperator):
```

Comment:
This seems like a weird file to put this in, originally, this file was "we can take an arbitrary function and wrap it into a higher order op, for testing". The new class doesn't "wrap" in the conventional sense, it always calls checkpoint. Should this not have gotten its own file in `torch/_higher_order_ops`?